examples/benchmarks/bubblebench/common/CHANGELOG.md

```diff
@@ -1,81 +0,0 @@
-# @fluid-example/bubblebench-common
```

Comment:
I think we can leave this fiel alone. We don't want to copy it's content into the new package, but we don't want to remove it from the old one.